### PR TITLE
[7.x.x] Update windows image for smoke tests

### PR DESCRIPTION
**Pull request by sophiaaar**
### Purpose of this PR
Fix the crash that was occurring for windows playmode smoke tests

The crash log had a lot of references to `OpenAdapter12`, and googling showed that this could be caused by out of date drivers. I switched the agents to use a different windows image because the windows version on the new image has newer drivers.

---
### Testing status
* Tested locally on mac, couldnt reproduce
* Tested on yamato https://unity-ci.cds.internal.unity3d.com/job/11196858

---
### Comments to reviewers
* The project updated when I opened it locally, hence the extra files.
* Cancelled the smoke test part of the general checks pipeline because it already ran here https://unity-ci.cds.internal.unity3d.com/job/11201702/dependency-graph

### CODE REVIEW

File: .yamato/smoke_test-win-dx11.yml [hunk 1]
```diff
@@ -41,8 +41,8 @@ Smoke_Test_Win_DX11_playmode_mono_Linear_2019.4:
 Smoke_Test_Win_DX11_editmode_mono_Linear_2019.4:
     name: Smoke_Test on Win_DX11_editmode_mono_Linear on version 2019.4
     agent:
-        type: Unity::VM
-        image: sdet/gamecode_win10:stable
+        type: Unity::VM::GPU
```
Comment (theo-pnv) [inline .yamato/smoke_test-win-dx11.yml:44]: Should we remove the GPU tag for editmode here?

File: .yamato/smoke_test-win-dx11.yml [hunk 2]
```diff
@@ -105,9 +105,9 @@ Smoke_Test_Win_DX11_Standalone_mono_Linear_2019.4:
 Build_Smoke_Test_Win_DX11_Standalone_mono_Linear_2019.4:
     name: Build Smoke_Test on Win_DX11_mono_Linear_Standalone_build_Player on version 2019.4
     agent:
-        type: Unity::VM
-        image: sdet/gamecode_win10:stable
-        flavor: b1.xlarge
+        type: Unity::VM::GPU
```
Comment (theo-pnv) [inline .yamato/smoke_test-win-dx11.yml:108]: Same

File: TestProjects/SRP_SmokeTest/Packages/manifest.json [hunk 3]
```diff
@@ -56,6 +49,7 @@
     "com.unity.modules.wind": "1.0.0",
     "com.unity.modules.xr": "1.0.0"
   },
+  "disableProjectUpdate": true,
```
Comment (theo-pnv) [inline TestProjects/SRP_SmokeTest/Packages/manifest.json:52]: Is this intended?

**Review by theo-pnv — CHANGES_REQUESTED** [review]

File: TestProjects/SRP_SmokeTest/Packages/manifest.json (hunk 3 above)
Comment (sophiaaar) [reply TestProjects/SRP_SmokeTest/Packages/manifest.json:52]: ah thanks

**Review by theo-pnv — APPROVED** [review]

**Pull request merged**